stable_baselines/common/vec_env/subproc_vec_env.py

```diff
@@ -107,3 +111,8 @@ def get_images(self):
             pipe.send(('render', {"mode": 'rgb_array'}))
         imgs = [pipe.recv() for pipe in self.remotes]
         return imgs
+
+    def env_method(self, method_name, *method_args, **method_kwargs):
+        for remote in self.remotes:
+            remote.send(('env_method',(method_name, method_args, method_kwargs)))
+        return np.stack([remote.recv() for remote in self.remotes])
```

Comment:
Here you assume that the data sent by the environments can be stack as numpy arrays, which is not always true. It's ok but should be clearly stated in the documentation.

Reply:
Ok.  In what file should this documentation appear?

Reply:
I meant in the docstring